docker/compose.devcontainer.yaml

```
@@ -161,11 +161,28 @@
+    volumes:
+      - source: redis_test_data
+        target: /data
+        type: volume
 volumes:
   api_bash_history:
   api_node_modules:
   api_pnpm_store:
   cloudbeaver_data:
   minio_test_data:
   postgres_test_data:
+  redis_test_data:
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)